### PR TITLE
Production controller file added

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: .github/workflows/codecov.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 name: Codecov Report Upload
+8d8a1b5
+bcb7e79
```
Comment (ashwin1111) [inline .github/workflows/codecov.yml:3]: remove these

File: deployment/production/controller.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: intacct-api-qcluster
+    spec:
+      containers:
+        - name: intacct-api
+          image: docker.io/sivafyle/fyle_intacct-api:vbcb7e79
```
Comment (Shwetabhk) [inline deployment/production/controller.yml:80]: fix docker hub username and version

File: deployment/production/controller.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: intacct-api
+    spec:
+      containers:
+        - name: intacct-api
+          image: docker.io/sivafyle/fyle_intacct-api:vbcb7e79
```
Comment (Shwetabhk) [inline deployment/production/controller.yml:36]: fix docker hub username and version

Comment (ashwin1111) [inline deployment/production/controller.yml:36]: image tag shouldn't be hard coded

File: deployment/production/controller.yml (hunk 2 above)
Comment (ashwin1111) [inline deployment/production/controller.yml:80]: same

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**